packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -463,6 +464,92 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     });
   }
 
+  @override
+  Future<List<Duration>> handlePointerEventPack(List<PointerEventPacket> records) {
+    assert(records != null);
+    assert(records.isNotEmpty);
+    return TestAsyncUtils.guard<List<Duration>>(() async {
+      // hitTestHistory is an equivalence of _hitTests in [GestureBinding]
+      final Map<int, HitTestResult> hitTestHistory = <int, HitTestResult>{};
+      final List<Duration> handleTimeStampDiff = <Duration>[];
+      DateTime startTime;
+      for(final PointerEventPacket packet in records) {
+        final DateTime now = binding.clock.now();
+        startTime ??= now;
+        // So that the first event is promised to received a zero timeDiff
+        final Duration timeDiff = packet.timeStamp - now.difference(startTime);
+        if (timeDiff.isNegative) {
+          // Flush all past events
+          // maybe trigger a warning
```

Comment:
Add a warning here and remove "// maybe trigger a warning". No one is probably going to add warnings or paying attention to this if this is just a comment.

Reply:
I believe it's no longer needed since the timediff is returned.